packages/frontend/pages/index.tsx

```diff
@@ -518,7 +518,7 @@ function TradePage() {
 
   useEffect(() => {
     setVolMultiplier(1.2)
-  }, [tradeType])
+  }, [])
```

Comment:
why remove this dep?

Reply:
It seems like it's not being used since the volMultiplier is always being set to 1.2 here

Reply:
if there's nothing in the dependency array does that mean it will always re-render or am I mistaken? Believe it was originally there because the Short side may have had a different vol multiplier, but we don't show a short side backtest right now 

We actually might not need this setter at all, think in the long backtest code EJ has it set at 1.2, but cc @yichiehliu to confirm?

Reply:
No, it will only be set with the initial render if there's nothing in the array. But I removed it now after asking @yichiehliu.